Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM golang:1.17 as builder
+FROM --platform=$BUILDPLATFORM golang:1.17 as builder
```

Comment:
We do not need this as we are doing cross-build using go.

Reply:
multi-arch runs won't build without it